### PR TITLE
Bump package versions to 0.3

### PR DESCRIPTION
**Pull request by nnmm**

### CODE REVIEW

**Comment by nnmm** [comment]
I'm a bit unsure whether it makes a difference to specify dependencies as "0.3" instead of "0.3.0". I used "0.3" for now, so that we don't need to update in case we do a backward-compatible patch release.

File: examples/message_demo/Cargo.toml [hunk 1]
```diff
@@ -12,13 +12,13 @@ path = "src/message_demo.rs"
...
-version = "*"
+version = "0.3"
 
 [dependencies.rosidl_runtime_rs]
-version = "*"
+version = "0.3"
 
 [dependencies.rclrs_example_msgs]
-version = "*"
+version = "0.3"
```
[the remark this answers is not in the document]
Comment (nnmm) [reply examples/message_demo/Cargo.toml:21]: No, this also works.

[the remark this answers is not in the document]
Comment (nnmm) [reply examples/message_demo/Cargo.toml:21]: I think (though I'm not sure) that the dependency version required here must match the crate found at the patch path. That's the case for `"*"` and `"0.3"`.

**Review by esteve — APPROVED** [review]

**Comment by nnmm** [comment]
Yeah :D

**Pull request merged**